python/ray/rllib/train.py

```diff
@@ -56,6 +56,12 @@ def create_parser(parser_creator=None):
         default=None,
         type=int,
         help="Emulate multiple cluster nodes for debugging.")
+    parser.add_argument(
+        "--ray-redis-max-memory",
+        default=None,
+        type=int,
+        help="--redis-max-memory to pass to Ray."
+        " This only has an affect in local mode.")
```

Comment:
Hi @ericl What does "local mode" mean? Does this work in multi-node mode?

Reply:
It just means that when using a cluster, you need to pass --redis-max-memory to `ray start` and not train.py